xbmc/platform/posix/filesystem/PosixFile.cpp

```diff
@@ -331,7 +334,39 @@ int CPosixFile::Stat(const CURL& url, struct __stat64* buffer)
   if (filename.empty() || !buffer)
     return -1;
 
+#if defined(HAS_STATX)
+  int dirfd = AT_FDCWD;
+  int flags = AT_SYMLINK_NOFOLLOW;
+  unsigned int mask = STATX_ALL;
```

Comment:
```suggestion
  unsigned int mask = STATX_BASIC_STATS | STATX_BTIME;
```

no need to request stuff in the future that we don't use (and might need extra time to fetch)

Reply:
I just wasn't sure if there were other things that might be needed elsewhere which is why I fetched everything.  But if that's not the case then I'm happy to change it.

Reply:
You only ever use the information in this function, so it is fine to only request the stuff that is actually used here